octavia/common/utils.py

```diff
@@ -62,13 +63,15 @@ def get_amphora_driver():
 
 
 def get_network_driver():
-    CONF.import_group('controller_worker', 'octavia.common.config')
-    network_driver = stevedore_driver.DriverManager(
-        namespace='octavia.network.drivers',
-        name=CONF.controller_worker.network_driver,
-        invoke_on_load=True
-    ).driver
-    return network_driver
+    global _NETWORK_DRIVER
```

Comment:
Looks good to me. But would a simple `@MEMOIZE` annotation not work here?
(Although that needs too much boilerplate code..)

Reply:
This is a singleton class object, I would rather not try to serialize it to memcache. the @MEMOIZE decorator is good for functions that take and return primitive datatypes.